Build break due to golang version on RPi

**Describe the bug**
[RPi instruction](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/doc/platforms/raspberry_pi3/raspberry_pi3.md#2-build-directly-on-the-raspberry-pi-3-board) guides developers to install golang with command `sudo apt install golang`.
RPi install 1.11 version of golang and it fails to build the project.
Find the log below.
```
pi@raspberrypi:~/edge-home-orchestration-go $ go version
go version go1.11.6 linux/arm
pi@raspberrypi:~/edge-home-orchestration-go $ ./build.sh

-----------------------------------
 Install prerequisite packages
-----------------------------------
(1/7) go get -u github.com/axw/gocov/gocov: Done
(2/7) go get -u github.com/matm/gocov-html: Done
(3/7) go get -u golang.org/x/lint/golint: Done
(4/7) go get -u github.com/Songmu/make2help/cmd/make2help: Done
(5/7) go get -u golang.org/x/mobile/cmd/gomobile# golang.org/x/tools/go/packages
../go/src/golang.org/x/tools/go/packages/golist_overlay.go:547:7: undefined: strings.ReplaceAll

download fail
```

--- a/doc/platforms/raspberry_pi3/raspberry_pi3.md
+++ b/doc/platforms/raspberry_pi3/raspberry_pi3.md
@@ -57,10 +57,13 @@ $ newgrp docker
 > For [execution of docker commands with non-root privileges](https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user) you need to add `$USER` to docker group.  
 `$ sudo usermod -aG docker $USER`
 
-- go compiler
+- go compiler (install a version not lower than 1.12.5, recommended v1.14.x)
 
 ```sh 
-$ sudo apt install golang
+$ wget https://dl.google.com/go/go1.14.5.linux-armv6l.tar.gz
+$ sudo tar -C /usr/local -xvf go1.14.5.linux-armv6l.tar.gz
+$ export GOPATH=$HOME/go
+$ export PATH=$PATH:$GOPATH/bin:/usr/local/go/bin
 ```
 
 - glide ([package manager for Go](https://glide.readthedocs.io/en/latest/))  

